envisage/ui/tasks/tests/test_tasks_application.py

```diff
@@ -47,10 +42,6 @@ class DummyGUI(HasTraits):
     pass
 
 
-@unittest.skipIf(
```

Comment:
This skip is redundant with the test-by-test skips applied to individual methods.